book/src/build_and_install.md

```diff
@@ -0,0 +1,28 @@
+# Building and Installing
+
+## Prerequisites
+To be able to build the source code, you will need to [install Rust](https://www.rust-lang.org/tools/install)
+and LLVM 11 along with the standard build tools (`build-essential`) and `liblz-dev` on your machine. 
```

Comment:
It  looks like on WSL at least, liblz-dev was not enough, I had to also install liblz1g-dev. Looking further into it, the actual metapackage on ubuntu is 'libz-dev' and not 'liblz-dev' (Extra l)
I can fix this up quick and commit though, no need to recommit on your side.

Reply:
Alright thanks!